src/test/groovy/tests/TestSetup.groovy

```diff
@@ -0,0 +1,16 @@
+package tests
+
+import org.yaml.snakeyaml.Yaml
+
+class TestSetup {
+    Yaml yaml = new Yaml()
+    protected final configFile = new File("./src/test/resources/testenv.yaml")
+    protected final config = yaml.load(configFile.text)
```

Comment:
You know, because you only load this config once, you cold/should change this line to be:
```groovy
protected static final Map config = new Yaml().load(new FileReader("./src/test/resources/testenv.yaml"))
```
Make sure you move it to another section of the class (top) to make it stand out as it has totally different qualifiers.

Reply:
done